c7n/resources/rdscluster.py

```diff
@@ -658,3 +660,44 @@ def process(self, resources, event=None):
             if expected_dates.issubset(snapshot_dates):
                 results.append(r)
         return results
+
+
+@RDSCluster.filter_registry.register('db-cluster-parameter')
+class ClusterParameterFilter(ParameterFilter):
+    """
+    Applies value type filter on set db cluster parameter values.
+    :example:
+    .. code-block:: yaml
+            policies:
+              - name: rdscluster-pg
+                resource: rds-cluster
+                filters:
+                  - type: db-cluster-parameter
+                    key: someparam
+                    op: eq
+                    value: someval
+    """
+    schema = type_schema('db-cluster-parameter', rinherit=ValueFilter.schema)
+    schema_alias = False
+    permissions = ('rds:DescribeDBInstances', 'rds:DescribeDBParameters',)
+    policy_annotation = 'c7n:MatchedDBClusterParameter'
+    paramcache = {}
+
+    def _get_para_list(self, pg):
+        client = local_session(self.manager.session_factory).client('rds')
+        paginator = client.get_paginator('describe_db_cluster_parameters')
+        param_list = list(itertools.chain(*[p['Parameters']
+            for p in paginator.paginate(DBClusterParameterGroupName=pg)]))
+        return param_list
+
+    def process(self, resources, event=None):
+        results = []
+        parameter_group_list = {db['DBClusterParameterGroup'] for db in resources}
```

Comment:
Can we add `param_group_attribute` to the class and then simplify the process method in the parent class to the following and remove the process method in this action:

```suggestion
        parameter_group_list = {db.get(self.param_group_attribute) for db in resources}
```

Reply:
updated ...please validate